### PR TITLE
New Chaplain Weapon: Healing Hand!

### PR DESCRIPTION
**Pull request by ghost**
You heal other people, it costs you cloneloss. I'm really not sure about the exact values yet though so how about you guys decide. I'll be adding a healing hand sprite for this when I can (tuesday!) but I want to make sure the code works.

### CODE REVIEW

**Comment by KorPhaeron** [comment]
Why would I use something that gives me the most difficult to remove damage type to heal the easiest to treat damage type

Especially when I spawn with a book that already heals that damage type and it does so without killing me

**Comment by DoctoreMad** [comment]
What KorPhaeron just said. But then, how would one make this work?

**Comment by KorPhaeron** [comment]
You can't fix it short of completely overhauling medical so that healing is rare and expensive enough that it'd be worth the tradeoff

**Comment by Armhulen** [comment]
I don't balance well, what do you think would be a good idea

**Comment by Armhulen** [comment]
Alternatively, toxin damage?

**Comment by Armhulen** [comment]
You know what would be neat just moving all damage to the chaplain

**Comment by DoctoreMad** [comment]
Well, we could change the concept a bit. Maybe give you a bonus the more selfless you are?

**Comment by AnonymousNow** [comment]
I've wanted the Healing Hands mutation for some time, but cloneloss? Not like this.

Why not make it take time and require a grab?

**Comment by FantasticFwoosh** [comment]
This also applies onto the pride thing but these designs are all too game focused (almost considered metagamey) and basically doubles up as a 100% heal chance for malus second bible like @KorPhaeron alludes to.

Just hit people with your bible, its balanced; what is your problem.

**Comment by Armhulen** [comment]
too game focused???????????

sawrge was right?

**Comment by Armhulen** [comment]
anyway i'm reworking this to take all their damage and put it on you, messiah's hand or something like that.

**Comment by DoctoreMad** [comment]
>stop playing combat-chaplain

What a **bore** you are. I will smite you the most.

**Comment by Armhulen** [comment]
>not giving your life to insta-heal the hos during battle

okayyyy

**Comment by ExcessiveUseOfCobblestone** [comment]
You already trade off your really good combat weapon for a thing that heals people that you can't remove?????

**Comment by Armhulen** [comment]
cobby tell me what you think would be fair

**Comment by ExcessiveUseOfCobblestone** [comment]
This sounds like a cool Druid esque item so perhaps make it like a range 2 medigun that heals toxin damage and replaces toxins with earthsblood so they can worship the trees too (making them extremely retarded but hey at least they don't have morphine haha)!

Bonus points if you make the beam green and nature oriented hehe

**Comment by Armhulen** [comment]
so it only heals them if they have toxins hmmm INTERESTING!

**Comment by DoctoreMad** [comment]
All checks have failed
1 failing check

**Comment by Armhulen** [comment]
When you only have the weebitor you gotta just shotgun out changes and see what Travis doesn't like

File: code/game/objects/items/holy_weapons.dm [hunk 1]
```diff
@@ -58,6 +58,33 @@
...
 	damtype = BURN
 	attack_verb = list("punched", "cross countered", "pummeled")
+	
+/obj/item/nullrod/healhand
+	icon_state = "disintegrate"
+	item_state = "disintegrate"
+	name = "hand of healing"
+	desc = "This hand of yours glows with a medical power!"
+	force = 1
+	flags_1 = ABSTRACT_1 | NODROP_1 | DROPDEL_1
```
Comment (jughu) [inline code/game/objects/items/holy_weapons.dm]: flags_1

Comment (ExcessiveUseOfCobblestone) [reply code/game/objects/items/holy_weapons.dm]: what are you saying

File: code/game/objects/items/holy_weapons.dm [hunk 2]
```diff
@@ -58,6 +58,33 @@
...
+	w_class = WEIGHT_CLASS_HUGE
+	hitsound = 'sound/weapons/sear.ogg' //remind me to add healing sounds :)
+	damtype = BURN
+	attack_verb = list("healed", "mended", "revitalized")
+
+/obj/item/nullrod/healhand/afterattack(atom/A, mob/user, proximity)
+	if(!proximity)
+		return
+	getBruteLoss(A)
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/holy_weapons.dm]: need to check if it's a mob first which is why it's not compiling.

File: code/game/objects/items/holy_weapons.dm [hunk 3]
```diff
@@ -58,6 +58,33 @@
...
+	force = 1
+	flags_1 = ABSTRACT_1 | NODROP_1 | DROPDEL_1
+	w_class = WEIGHT_CLASS_HUGE
+	hitsound = 'sound/weapons/sear.ogg' //remind me to add healing sounds :)
+	damtype = BURN
+	attack_verb = list("healed", "mended", "revitalized")
+
+/obj/item/nullrod/healhand/afterattack(atom/A, mob/user, proximity)
+	if(!proximity)
+		return
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/holy_weapons.dm]: new line `if ismob(A)` new line `var/mob/healmepls = A`

File: code/game/objects/items/holy_weapons.dm (hunk 2 above)
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/holy_weapons.dm]: `healmepls.getBruteLoss()`

File: code/game/objects/items/holy_weapons.dm [hunk 4]
```diff
@@ -58,6 +58,33 @@
...
+	getBruteLoss(A)
+	user.adjustBruteLoss(brute/2)
+	A.adjustBruteLoss(-brute)
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/holy_weapons.dm]: healmepls instead of A

File: code/game/objects/items/holy_weapons.dm [hunk 5]
```diff
@@ -58,6 +58,33 @@
...
+	var/mob/living/carbon/human/H = A
+	if(!istype(H))
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/holy_weapons.dm]: replace 80+81 with `if(ishuman(A))` new line `var/mob/living/carbon/human/healmepls2 = A` then indent the things below and remove the return.

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
this should fix it but stop using the webbitor you nerdola

File: code/game/objects/items/holy_weapons.dm (hunk 5 above)
Comment (ExcessiveUseOfCobblestone) [reply code/game/objects/items/holy_weapons.dm]: oh and replace `H` with `healmeplsiamhuman`

Comment (Armhulen) [reply code/game/objects/items/holy_weapons.dm]: clown car names but i'll do it!

**Comment by Pascal125** [comment]
What if, instead of clone-loss, it gives the chaplain brain damage.
That way, people won't whine about at-least one of the two excuses people have when a chaplain is healing them. Which is. "I'm becoming dumb! REEEEE".

File: code/game/objects/items/holy_weapons.dm (hunk 5 above)
Comment (ExcessiveUseOfCobblestone) [reply code/game/objects/items/holy_weapons.dm]: I mean you can replace the var names with whatever but I prefer to make them blatantly obvious with a side of hilarity!

**Comment by Armhulen** [comment]
hmmmmmmmmmmmm this is gonna be a headache to fix, maybe i'll come back to it as a druidy channel thing

**Pull request closed without merging**